docs/source/conf.py

```diff
@@ -51,9 +77,6 @@
     "nbsphinx",
 ]
 
-nbsphinx_execute = "always"
```

Comment:
Should this be set to `nbsphinx_execute = "never"` instead?

Reply:
Fair point